src/app/return-credentials/gst-return/gst-return.component.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, OnInit } from '@angular/core';
+import {SyncupApiService} from 'src/app/shared/api/syncup-api.service';
+import {FormGroup, FormBuilder, FormArray, Validators} from '@angular/forms';
+
+@Component({
+  selector: 'app-gst-return',
+  templateUrl: './gst-return.component.html',
+  styleUrls: ['./gst-return.component.css']
+})
+export class GstReturnComponent implements OnInit {
+
+  /*private gstReturnForm: FormGroup;*/
+
+  private gstReturnForm: FormGroup;
+
+
+  constructor(private formBuilder: FormBuilder, private apiService: SyncupApiService) {
+      this.gstReturnForm = this.formBuilder.group({
+        returnForms: this.formBuilder.array([
+          this.getReturnForm()
+        ])
+      });
+   }
+
+  ngOnInit() {
+  }
+
+   getReturnForm(): FormGroup {
+     return this.formBuilder.group({
+       gstFlatNo: [null, [Validators.required]],
+       gstArea: [null, [Validators.required]],
+       gstCity: [null, [Validators.required]],
+       gstState: [null, [Validators.required]],
+       gstPin: [null, [Validators.required]],
+       gstNo: [null, [Validators.required]],
+       gstUserName: [null, [Validators.required]],
+       gstPassword: [null, [Validators.required]]
```

Comment:
use this way for validating form at other places

Reply:
will do in next PR